### ISSUE TITLE
a bug in cuda detectMultiScale

### ISSUE THREAD

**Issue by bigdimboom** [opencv/opencv#10952]
##### Detailed description

https://github.com/opencv/opencv/blob/cfe84b953cd4218df88144e1d58c2580540c6972/modules/cudaobjdetect/src/cascadeclassifier.cpp

an assertion bug in line 179

**Comment by bigdimboom** [comment]
Yeah, I'm pretty sure it's a bug. What's the procedure of submitting a bug-fixing request.

**Comment by berak** [comment]
@bigdimboom ,

> What's the procedure of submitting a bug-fixing request.

https://github.com/opencv/opencv/wiki/How_to_contribute

(also the [wiki](https://github.com/opencv/opencv/wiki) has a lot of good hints on git, etc. (look at the right sidebar)

**Comment by vishnukumarts** [comment]
+1

**Comment by alalek** [comment]
Does code work properly with assertion check removal (there are no no tests for this case)?

**Comment by vishnukumarts** [comment]
Nope. 
Even i Pass the stream all the GPU works running in default stream only.  Cross verified in nvidia profiler.
